build-logic/src/main/kotlin/kotlin-inject.publish.gradle.kts

```diff
@@ -51,6 +51,7 @@ publishing {
         val publishApple by tasks.registering {
             publications.all {
                 if (name.contains(Regex("macos|ios|tvos|watchos"))) {
+                    @Suppress("DEPRECATION")
                     dependsOn("publish${name.capitalize(Locale.ROOT)}PublicationToSonatypeRepository")
```

Comment:
This can be replaced with `name.replaceFirstChar { it.uppercase() }`

Reply:
True, I'm still a little salty that `capitalize` was deprecated. I'll make the change